### ISSUE TITLE
"End Create" Event

### ISSUE THREAD

**Issue by LesserGoldfish** [YoYoGames/GameMaker-Bugs#4983]
### Is your feature request related to a problem?

I honestly don't know how to justify this outside of the one niche issue I encountered: I am using a very basic entity component system and sometimes when I want a component struct to have a refrence to another one at its creation I have to be very careful in what order I initialise them, otherwise the one I am trying to access may not exist yet. I thought about adding an event to the components that is called after all the components are created, but ran into the issue of having to copy and paste that logic to run that event into every entity object becuase I'm already using event_inherited at the start of the event. So I thought it would be nice to a have an "end create" event for the niche cases like these

(Sorry if this is unclear, I have trouble putting complex thoughts into writing)

### Describe the solution you'd like

Basically how the "End Step" and "End Draw" events work but for the create event. I don't think a "begin create" event is necesseray.

**Comment by gnysek** [comment]
"create end" event would still fire in some order, causing it to execute before/after some instance(s) are created, causing similar issues (with exception that they already exists).

Simplest solution is to create those instances from another object, then execute code you wish knowing they already exists (and it allows to use `with()` in that case).

**Comment by backYard321** [comment]
I'm not against this _in principle_, but I will say that any time I've felt the need for something like this (including a similar case to OP) I've found a different solution that ended up being more elegant and that I was happier with.

**Comment by stuckie** [comment]
This is not something we are planning.
